### ISSUE TITLE
[BUG] Empty test page in HTML reporter when using test list

### ISSUE THREAD

**Issue by yuvalkarif** [microsoft/playwright#11830]
**Context:**
- Playwright Version: 1.17.2
- Operating System: Windows
- Node.js version: v16.13.0
- Browser: Chromium

**Code Snippet**

```javascript
 // test.list.js
require('../ageGate/index.spec.js')
require('../account/register/index.spec.js')
require('../account/login/index.spec.js')
require('../account/recovery/index.spec.js')

// playwright.config.js
const config = {...
testMatch: 'test.list.js',
reporter: [
	['json', { outputFile: 'report.json' }],
	['html', { open: 'never' }],
   ],
}
```

**Describe the bug**
As suggested in the docs I am using a test list to run my tests in a specific order,
the thing is when I open the html-reporter they all render fine but when I click to expand each test to its own page I get an empty page.
If I run each test group it will generate the page correctly with all of its features working.

**Comment by yury-s** [comment]
I tried a similar setup and it worked. Can you share complete example so that we can reproduce it locally?

**Comment by pavelfeldman** [comment]
Fixed as 3a5e8184b58d68b73f521cc28714e42b18a5159e
